polaris-react/src/components/Choice/Choice.scss

```
@@ -74,6 +83,10 @@
 .Descriptions {
   // stylelint-disable-next-line -- Polaris component custom properties
   --pc-choice-size: 20px;
+  #{$se23} & {
+    // stylelint-disable-next-line -- Polaris component custom properties
```

Is `PSE23` a standard / common identifier we're using for finding these lint rule disablings? If so, this one is missing:

```suggestion
    // stylelint-disable-next-line -- PSE23 Polaris component custom properties
```